src/Microsoft.Kusto.ServiceLayer/DataSource/ReliableDataSourceConnection.cs

```diff
@@ -111,44 +115,6 @@ public void Dispose(bool disposing)
         /// Gets or sets the connection string for opening a connection to the SQL Azure database.
         /// </summary>
         public string ConnectionString { get; set; }
-        
-        /// <summary>
-        /// Gets the policy which decides whether to retry a connection request, based on how many
-        /// times the request has been made and the reason for the last failure. 
-        /// </summary>
-        public RetryPolicy ConnectionRetryPolicy
```

Comment:
These are properties which come from the connection dialog and I think are used in other code in Microsoft.SqlTools.CoreServices. You can see these properties when you click on "Advanced Properties" in the connection dialog. We should test these and remove them if they are not needed.